wire-library/wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -112,9 +112,16 @@ class KotlinGenerator private constructor(
    * Returns the full name of the class generated for [service]#[rpc]. This returns a name like
    * `RouteGuideClient` or `RouteGuideGetFeatureBlockingServer`.
    */
-  fun generatedServiceName(service: Service, rpc: Rpc? = null): ClassName {
+  fun generatedServiceName(
+    service: Service,
+    rpc: Rpc? = null,
+    isImplementation: Boolean = false
```

Comment:
Do we care about breaking this class' API?

Reply:
Definitely not.